ivy/functional/backends/numpy/elementwise.py

```diff
@@ -20,6 +20,7 @@ def add(x1: np.ndarray, x2: np.ndarray) -> np.ndarray:
     if not isinstance(x2, np.ndarray):
         x2 = np.asarray(x2, dtype=x1.dtype)
     return np.add(np.asarray(x1), np.asarray(x2))
+add.unsupported_dtypes = tuple(['float16'])
```

Comment:
I just chose this one at random, and it seems as though `numpy.add` does support `float16`. This code runs without error:

```
import numpy as np
x = np.array([1.], dtype='float16')
y = np.array([2.], dtype='float16')
print(np.add(x, y))
```